roles/common/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+common_timezone: 'Etc/UTC'
+
+# encfs
+encfs_password: "{{ lookup('password', 'credentials/' + role + '/encfs_password' length=32') }}"
```

Comment:
How do you envision migrating existing passwords?  Also, did you consider using Ansible Vault?  The ansible docs for `lookup` suggested that Vault may be the better option.